src/provider/index.ts

```diff
@@ -22,6 +22,9 @@ import {Store} from './store/Store';
 import {EventPump} from './model/EventPump';
 import {ClientRegistry} from './model/ClientRegistry';
 import {Database} from './model/database/Database';
+import {Timer} from './utils/Timer';
+
+const POST_BLUR_TOGGLE_BLOCK_DURATION: number = 750;
```

Comment:
Don't think we ever had a formal consensus, but we have generally been leaving the type declaration off in cases like this where the type is pretty obviously inferred. If you think that's something that should change then we can maybe discuss.

As an aside, our old linter used to complain when we did this which is probably why we err on the side of not having the type.

Reply:
✅